examples/react-native-vanilla/__tests__/__snapshots__/storyshots.js.snap

```diff
@@ -97,77 +97,68 @@ exports[`Storyshots Button with text 1`] = `
 `;
 
 exports[`Storyshots Knobs with knobs 1`] = `
-<View
-  style={
-    Object {
-      "backgroundColor": "#ffff00",
-      "borderColor": "#ff00ff",
-      "borderWidth": 3,
-      "padding": 10,
-    }
-  }
->
+Array [
```

Comment:
Looks like this is NOT what we want to happen. Part of the actual story is gone from the snapshot!